tedana/decomposition/eigendecomp.py

```diff
@@ -185,6 +185,8 @@ def tedpca(catd, OCcatd, combmode, mask, t2s, t2sG,
         Boolean mask array
     ref_img : :obj:`str` or img_like
         Reference image to dictate how outputs are saved to disk
+    bf : :obj:`str`
```

Comment:
Can we rename this to something a little more intuitive ? You mean everything before like `echo-` in the original filename, right ?

Reply:
That part of the filename and the output directory as well. Do you have a preference for the new name? `bf` stands for base filename, so that's a possibility.